cloud-init/docker-only-with-persistent-storage.txt

```diff
@@ -0,0 +1,79 @@
+#cloud-config
+
+coreos:
+  units:
+    - name: docker-tcp.socket
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=Docker Socket for the API
+
+        [Socket]
+        ListenStream=2375
+        BindIPv6Only=both
+        Service=docker.service
+
+        [Install]
+        WantedBy=sockets.target
+
+    - name: format-ephemeral.service
```

Comment:
This and `persistent-vda.mount` can be replaced by mounting in a disk and passing `root=/dev/vda` or `root=LABEL=<disk-label>` in the kernel cmdline. The PXE image doesn't have a rootfs, and supports providing your own root disk.